### ISSUE TITLE
assert support is missing

### ISSUE THREAD

**Issue by faassen** [astorije/chai-immutable#6]
I tried to use this plugin but was confused why it didn't work for "assert.equal". It turns out you need to install a bridging function on the "assert" object, see this mailing list entry:

https://groups.google.com/forum/#!topic/chaijs/Dwnbsbe0C70

I tried the following:

```
assert.equal = function(act, exp, msg) {
    new chai.Assertion(act, msg).to.equal(exp);
};
```

and now it works. The same should be done with the other overrides.

**Comment by astorije** [comment]
Hey, thanks for trying to use the plugin, and for reporting this!

As you noticed, I just started this, it's not even on the npm repository yet!
So far I focused on expect support only as it was my direct need. I'll fix this soon and report on this issue.

**Comment by faassen** [comment]
Yes, I noticed you just started it, but I ran into it googling for chai integration with Immutable.js and I figured I'd give you the feedback. Thanks for looking into this!

**Comment by astorije** [comment]
And I thank you for your feedback :-)
I'll let you know as soon as I can guarantee more stability on this. Stay tuned!

**Comment by astorije** [comment]
Hey @faassen!

1bcfae6 adds support for `assert.equal`. Before that, `assert.strictEqual` and `assert.deepEqual` would have worked since they both use the overridden base assertions, but now it's consistent across TDD and BDD interfaces.
Could you try it on your side to confirm it works as you want? I am not publishing a new version into npm until you do so, so you have to install the package from GitHub instead of npm packages: `npm uninstall && npm install astorije/chai-immutable`.

I am leaving this issue open until I add support of `assert.sizeOf`. After that I think I can close this issue. Do you see another assertion we could benefit from in the TDD interface?

**Comment by astorije** [comment]
Actually, adding support for `assert.sizeOf` was really straightforward so I did it right after, so that when you confirm you're okay with this I can publish into npm.

I close the issue because it seems to me that everything is covered for now. I'll reopen if you tell me that it doesn't match your expectations or doesn't work or needs more assertions.

**Comment by astorije** [comment]
Well actually (bis), I just published this support in version `0.3.0`. My questions are still valid, but that'd be great if you can open a new issue instead of re-opening this one.

Thanks!
